src/base64.cr

```diff
@@ -290,7 +290,7 @@ module Base64
   # *input* must have at least `pairs * 3` bytes available to process,
   # while *output* must have at least `pairs * 4` bytes of storage available.
   private def encode_base64_full_pairs_internal(input : UInt8*, output : UInt8*, pairs : Int32, chars : UInt8*) : Nil
-    while pairs >= 8
+    while pairs > 8
```

Comment:
what is this fix? why no specs failed before it? no spec for this case?

Reply:
I think it would be helpful if that summary were added directly to the code as a comment for future readers.